src/sections/what-is-service-mesh/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { MDXRenderer } from "gatsby-plugin-mdx";
+import { Link } from "gatsby";
+
+import bookImage from "../../collections/books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition/The-Enterprise-path-to-service-mesh-Architectures-2nd-Edition.png";
+import ServiceMeshWrapper from "./whatServiceMesh.style";
+
+const WhatServiceMesh = ({gradient}) => {
+    if(gradient === undefined)
+        gradient = true;
+    return (
+        <ServiceMeshWrapper gradient={gradient}>
+            <h1>What Is A Service Mesh?</h1>
+            <br/>
+            <span className="description">A dedicated layer for managing service-to-service communication.</span>
```

Comment:
@OmkarPh great work here. Just one thing, Can the font-weight be decreased a bit(make it lighter)?

Reply:
Actually I didn't do any changes in code, the package-lock.json was unexpectedly showing changes, may be some issue with line endings, I just rebased my branch, will do actual changes soon

Reply:
Yaa, I noticed that afterwards😂

Reply:
😂✌️

Reply:
Any other changes?
Some specific styles to be given to heading or image resolution ...

Reply:
The image takes a lot to load currently.

Reply:
The other option we have is to use thubnail, https://deploy-preview-1213--layer5ng.netlify.app/static/0190594eefcb3e3c29c604706bdd6184/0d68b/The-Enterprise-path-to-service-mesh-Architectures-2nd-Edition-small.webp 

I don't think thumbnail would look good here

Reply:
Is font weight ok?
You need gradient prop for the toggle right?